City-of-Helsinki/drupal-helfi-platform-config
# UHF-9312 unnecessary config_filter in composer.json

# [UHF-9312](https://helsinkisolutionoffice.atlassian.net/browse/UHF-9312)
Cleaning up platform_config composer.json


## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-9312`
* Run `make drush-updb drush-cr`


## How to test
- config_filter should have been removed
  - It is not directly used, it is dependency for config ignore-module. 
- run drush cex
  - core.extension should have been changed.
  - hal and rdf modules should have been disabled if in use.
